aspnetcore/tutorials/razor-pages/new-field.md

```diff
@@ -42,10 +42,11 @@ Automatic verification that the schema and model are in sync makes it easier to
 
    [!code-cshtml[](razor-pages-start/sample/RazorPagesMovie50/SnapShots/IndexRating.cshtml?highlight=40-42,62-64)]
 
-1. Update the following pages:
-   1. Add the `Rating` field to the Delete and Details pages.
-   1. Update [Create.cshtml](https://github.com/dotnet/AspNetCore.Docs/tree/main/aspnetcore/tutorials/razor-pages/razor-pages-start/sample/RazorPagesMovie50/Pages/Movies/Create.cshtml) with a `Rating` field.
-   1. Add the `Rating` field to the Edit Page.
+1. Update the following pages with a `Rating` field:
+   1. *[Pages/Movies/Create.cshtml](https://github.com/dotnet/AspNetCore.Docs/tree/main/aspnetcore/tutorials/razor-pages/razor-pages-start/sample/RazorPagesMovie30/Pages/Movies/Create.cshtml)*.
```

Comment:
It's this the 6.0 version? Check the moniker at the top. If it's not 6, update the 6.0 version too. You can do that in another PR.

Reply:
No, the first commit was for version 5.0 only. But I updated other sections too which cover version < 5.0 and version 6.0.